shared/checks/oval/partition_for_tmp.xml

```diff
@@ -13,7 +13,7 @@
       <criterion test_ref="test_tmp_partition" comment="/tmp on own partition" />
     </criteria>
   </definition>
-  <linux:partition_test check="all" check_existence="all_exist"
+  <linux:partition_test check="all" check_existence="none_exist"
```

Comment:
Can you provide more detail on why this change? Check will pass now if `/tmp` does not exist.